src/main/java/net/fabricmc/loom/api/processor/ProcessorContext.java

```diff
@@ -41,5 +41,5 @@ public interface ProcessorContext {
 
 	LazyCloseable<TinyRemapper> createRemapper(MappingsNamespace from, MappingsNamespace to);
 
-	MemoryMappingTree getMappings();
+	MemoryMappingTree getMappings(String srcNs);
```

Comment:
Why do you need to specity the source ns everwhere? I woudl generally expect it to be intermediary, is this not the case?

Reply:
It was added because the impl calls `MappingConfiguration.getMappingsService` which now asks for the source namespace. But you're right, it's not needed as a parameter here, and can just be hardcoded to intermediary in `ProcessorContextImpl`.